### DISCUSSION TITLE
Material Icons, how to set up?

### DISCUSSION THREAD

**Discussion by craigs100** [skartknet/Iconic#31] [Q&A]
Using the pre-configured option for Material,  as far as I understand I should be able to point the CSS file and Routes file at https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined and it should work, but it doesn't seem to. Have tried regexs of `"(\w+)":\s"[^"] `& `"(\w+)":"[^"]`.  I get errors of "Invalid Configuration: Error loading CSS file." and "Invalid Configuration: The extracted icons are 0. Please check that the Selector regex is correct" when I hit the "Reload Preview button".  Using Umb 11.2.2 and Iconic 5.2.1. Any advice would be appreciated.

**Comment by craigs100** [comment]
I had some success with the instructions here: https://github.com/skartknet/Iconic/tree/master/Documentation/Configuration/MaterialIcons  which allowed me to get a property editor working in a Block List block.  However, the instructions for rendering on the front end here: https://github.com/skartknet/Iconic/tree/master/Documentation/Usage don't appear to work. @Html.Raw(Model.MyIconPicker) doesn't work. I get an error of: 
`JsonSerializationException: Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'Our.Iconic.Core.Models.SelectedIcon' because the type requires a JSON object (e.g. {"name":"value"}) to deserialize correctly.`

Also I don't have @using Iconic.Helpers available in Views either so I appear to be stuck.
Any advice would be appreciated.

**Reply by skartknet** [reply]
Hi @craigs100 ,

I have followed the instructions myself configuring a brand new Umbraco v11.2.2 and Iconic 5.2.2 and it's working fine. 

Are you upgrading the package from a previous version or it's a new installation?

**Reply by skartknet** [reply]
Also I was able to use the `Html.RenderIcon` helper. But it's on the `Our.Iconic.Core.Helpers` namespace. It seems you are using an old namespace?

**Reply by craigs100** [reply]
Thanks for the reply. This is a first install, not an upgrade. 

It says: "For this extension to work you'll have to include @using Iconic.Helpers on your view." on this page: https://github.com/skartknet/Iconic/tree/master/Documentation/Usage

Trying with the one you specified, I can now output Material and FontAwesome 4 icons on the same page :)  Any plans to do FontAwesome 6? I think they're the ones I really want, lol

**Comment by skartknet** [comment]
Hi @craigs100 , did you have any luck with installing FA 6?

**Reply by craigs100** [reply]
IIRC I removed Iconic and the client supplied some custom SVG's instead which I hard coded in. Not as flexible but got the job done. Haven't had a similar requirement since, I'm afraid.
